src/lib/types/settings/ClientSettings.ts

```diff
@@ -4,8 +4,8 @@ import { T } from './Shared';
 export namespace ClientSettings {
 
 	export namespace Blacklist {
-		export const Guilds = T<string[]>('blacklist.guilds');
```

Comment:
Keep this schema; change in `src/commands/bot-owner/blacklist.ts` to match to prior commit of this file.